sigar_linux.go

```diff
@@ -388,3 +418,15 @@ func readProcFile(pid int, name string) ([]byte, error) {
 
 	return contents, err
 }
+
+/* Compare major device ID to registered block device IDs from the kernel */
+func isBlockDevice(majorDevId uint64) bool {
```

Comment:
Can we expand the context of this function a little?  I think I understand its intent, but I'm not sure I understand the device id's being checked.